include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3329,6 +3331,38 @@ struct OrtApi {
   ORT_API2_STATUS(AddExternalInitializers, _In_ OrtSessionOptions* options,
                   _In_reads_(input_len) const char* const* initializer_names,
                   _In_reads_(input_len) const OrtValue* const* initializers, size_t initializers_num);
+
+  ORT_API2_STATUS(CreateAttribute,
+                  _In_ const char* name,
+                  _In_ const void* data,
+                  _In_ int len,
+                  _In_ ONNXTensorElementDataType type,
+                  _In_opt_ int is_array,
+                  _Outptr_ OrtOpAttr* op_attr);
+
+  ORT_API2_STATUS(ReleaseAttribute, _Inout_ OrtOpAttr* op_attr);
```

Comment:
ORT_CLASS_RELEASE ?

Reply:
Yes - prefer to nullify the pointer to minimize the chance of misuse after release.

Reply:
But isn't it inconsistent with other exposed Release() APIs ?